Formula/crystal.rb

```diff
@@ -63,13 +101,16 @@ def install
                           "src/compiler/crystal.cr",
                           "--release", "--no-debug"
 
+    # Build shards
     resource("shards").stage do
       system buildpath/"bin/crystal", "build", "-o", buildpath/".build/shards", "src/shards.cr"
     end
 
     bin.install ".build/shards"
     bin.install ".build/crystal"
     prefix.install "src"
+    (prefix/"embedded/lib").install "#{@bdwgc_path}/.libs/libgc.a"
```

Comment:
I thought the idea was that `libgc` is statically linked but not installed…

Reply:
Although it is linked in the crystal binary the goal is to have it available for linking in the programs compiled with Crystal. That is the reason why it is installed.